net.c

```diff
@@ -865,7 +865,7 @@ void net_process_return(void)
   struct sockaddr * fromsockaddr = (struct sockaddr *) &fromsockaddr_struct;
   struct sockaddr_in * fsa4 = (struct sockaddr_in *) &fromsockaddr_struct;
   socklen_t fromsockaddrsize;
-  int num;
```

Comment:
Would it clean up further, do you think, if num would be "size_t", and that you cast just when doing the recvfrom?